src/Configuration.php

```diff
@@ -548,4 +561,42 @@ static function (array $files, SplFileInfo $fileInfo): array {
             []
         );
     }
+
+    /**
+     * @param array $config
+     *
+     * @return string
+     */
+    private static function retrieveOnExistingOutputDir(array $config): string
+    {
+        if (false === array_key_exists(self::ON_EXISTING_OUTPUT_DIR_KEYWORD, $config)) {
+            return 'ask';
+        }
+
+        $onExistingOutputDir = $config[ self::ON_EXISTING_OUTPUT_DIR_KEYWORD ];
+
+        if(!is_string($onExistingOutputDir)) {
+            throw new InvalidArgumentException(
+                sprintf(
+                    'Expected %s to be a string, found "%s" instead.',
+                    self::ON_EXISTING_OUTPUT_DIR_KEYWORD,
+                    gettype( $onExistingOutputDir )
+                )
+            );
+        }
+
+        $allowedOnExistingOutputDirValues = ['ask', 'overwrite', 'abort'];
+        if(!in_array($onExistingOutputDir, $allowedOnExistingOutputDirValues)) {
+            throw new InvalidArgumentException(
+                sprintf(
+                    'Found value of "%s" for %s; allowed values are %s.',
```

Comment:
```suggestion
                    'Expected one of "%s" for "%s". Got "%s"',
```

Careful you need to rearrange the order the parameters with this suggestion